lib/main_common.pm

```diff
@@ -534,8 +545,13 @@ sub load_system_role_tests {
         loadtest "installation/logpackages";
     }
     loadtest "installation/disable_online_repos" if get_var('DISABLE_ONLINE_REPOS') && !get_var('OFFLINE_SUT');
-    loadtest "installation/installer_desktopselection" if is_opensuse;
-    loadtest "installation/system_role" if is_caasp('kubic');
+    if (is_leap('<=15.0')) {
+        loadtest "installation/installer_desktopselection";
+    }
+    elsif (is_opensuse || is_sle12sp2_using_system_role() || is_sle('15+'))
```

Comment:
I guess this flow will be used in TW only as a start

Reply:
so is_opensuse is enough for now?

Reply:
My thought would be to just use `else` but now I am confused because of SLE cases where we do not expect the system role ?:-(

Reply:
The module is not scheduled for those conditions in SLE12, for example: [sle-12-SP4-**Server**-DVD-x86_64-Build0327-gnome@64bit ](https://openqa.suse.de/tests/1902202#step/system_role/1) vs [sle-12-SP4-**Desktop**-DVD-x86_64-Build0276-gnome@64bit ](https://openqa.suse.de/tests/1901439#step/partitioning/1)

Reply:
leap 15.1 won't have system roles yet, so on top of old behavior, we should only include TW. Also we should put is_caasp('kubic') as it was selecting test module before.